platformsh-flex-env.php

```diff
@@ -142,3 +145,34 @@ function mapPlatformShDatabase() : string
 
     return $dbUrl;
 }
+
+function mapPlatformShMongoDatabase() : void
+{
+    // Set the MONGODB_SERVER, MONGODB_DB, MONGODB_USERNAME, and MONGODB_PASSWORD
+    // for Doctrine, if necessary.
+    // full dsn is not valid, so we need those 4 env variables.
```

Comment:
"full dsn is not valid".  This is not a complete sentence.  "The Full DSN is not valid (give some reason why?), so populate all 4 environment variables.)

Reply:
I made changes